.travis.yml

```diff
@@ -4,6 +4,8 @@ rvm:
   - 2.2.3
 install:
   - bundle install
+  - nvm install 5.0
+  - nvm use 5.0
   - npm install -g npm
```

Comment:
Hi @dylangrafmyre I just merged. Let's do another PR if that's the case.

BTW, I get the linting error locally, so this is probably not the main build issue. There's some problem with eslint?